api.py

```diff
@@ -1,68 +1,119 @@
-import hug
+import html
+import logging
+import os
+import time
+
 import falcon
+import geoip2.webservice
+import hug
 import maxminddb
-import time
-import logging
+from geoip2.errors import GeoIP2Error
+
+GEOIP2_USER_ID = os.getenv("GEOIP2_USER_ID", "")
+GEOIP2_LICENSE_KEY = os.getenv("GEOIP2_LICENSE_KEY", "")
+GEOIP_SERVICE = 2
+
+result = {
+    "ip": "",
+    "continent_code": "",
+    "continent_name": "",
+    "country_code": "",
+    "country_name": "",
+    "region_code": "",
+    "region_name": "",
+    "city": "",
+    "zip": "",
+    "latitude": "",
+    "longitude": "",
+    "location": {"geonameid": "", "time_zone": ""},
+}
 
-class GeoIP:
-    def __init__(self, reader):
-        self.reader = reader
 
-    @hug.object.get(urls='/geoip/{ipv4}')
+class GeoIP:
+    @hug.object.get(urls="/geoip/{ipv4}")
     def get(self, ipv4: str):
+        if GEOIP_SERVICE == 1:
```

Comment:
We will clean up this Part after it will be successful on the Live. Since this code doesnt have any Staging environment. I will delete also the part that GEOIP-non paid version gets CSV from Maxmind GEOIP to get the database on the clean up.